cek/cost_model.go

```diff
@@ -75,21 +75,15 @@ func sizeExMem(i int) func() ExMem {
 
 // Return a function so we can have lazy
 // costing of params for the case of constant functions
+//
+//go:inline
 func bigIntExMem(i *big.Int) func() ExMem {
 	return func() ExMem {
-		x := big.NewInt(0)
-
-		if x.Cmp(i) == 0 {
+		if i.Sign() == 0 {
 			return ExMem(1)
-		} else {
-			x := big.NewInt(0)
-
-			x.Abs(i)
-
-			logResult := x.BitLen() - 1
-
-			return ExMem(logResult/64 + 1)
 		}
+		logResult := i.BitLen() - 1
+		return ExMem(logResult/64 + 1)
```

Comment:
Can we avoid another allocation here by doing the `i.BitLen() - 1` inline with the `return`? Also, the `logResult` variable name doesn't seem very descriptive of what it's actually representing